ci/travis/script-run-tests.sh

```diff
@@ -28,7 +28,11 @@ echo
 ${TRAVIS_BUILD_DIR}/scripts/run_nupic_tests.py -u --coverage --failfast || exit
 
 mv ${TRAVIS_BUILD_DIR}/.coverage ${TRAVIS_BUILD_DIR}/.coverage_unit
-# Python integration tests and prep for coveralls reporting
-${TRAVIS_BUILD_DIR}/scripts/run_nupic_tests.py -i --coverage --failfast || exit
 
-mv ${TRAVIS_BUILD_DIR}/.coverage ${TRAVIS_BUILD_DIR}/.coverage_integration
+# Removed because integration tests take so long and we run them in
```

Comment:
I'd prefer to revert this change. We shouldn't rely on the regression suite to see if things break. I see it more as testing that we still get good NAB scores and prediction results and for perf testing. Our unit tests aren't sufficient for many breakages so we need integration tests too.